controls/V-73387.rb

```diff
@@ -81,7 +81,7 @@
   forrest = attribute('forrest')
```

Comment:
you should use the `input` directly in the control rather than setting a local instance variable.